dll/win32/rpcrt4/rpc_transport.c

```diff
@@ -274,6 +275,7 @@ static DWORD rpcrt4_create_pipe_security(PSECURITY_DESCRIPTOR *SecDesc)
     if (!MakeSelfRelativeSD(AbsSD, RelSD, &RelSDSize) && GetLastError() == ERROR_INSUFFICIENT_BUFFER)
```

Comment:
It looks odd to ignore other (potential) error values. The 3 other files do not do that.

Reply:
I'll recheck for any instance I may have missed.